README.md

```diff
@@ -51,12 +51,19 @@ For greenfield projects you might want to consider to use [mobx-react-lite](http
 
 Please check [mobx.js.org](https://mobx.js.org) for the general documentation. The documentation below highlights some specifics.
 
-### `observer(componentClass)`
+### `observer(component)`
 
-Function (and decorator) that converts a React component definition, React component class or stand-alone render function into a reactive component, which tracks which observables are used by `render` and automatically re-renders the component when one of these values changes.
```

Comment:
The part that says "used by `render`" no longer feels relevant to functional components, so instead I said "used by its effective `render`".  Still doesn't seem right and might need to elaborate.

Reply:
I think it's fine like that.